app/src/main/java/org/oppia/app/customview/inputInteractionView/NumericInputInteractionView.kt

```diff
@@ -0,0 +1,53 @@
+package org.oppia.app.customview.inputInteractionView
+
+import android.content.Context
+import android.text.InputFilter
+import android.text.method.DigitsKeyListener
+import android.util.AttributeSet
+import android.widget.EditText
+import org.oppia.app.R
+import org.oppia.app.model.InteractionObject
+
+/** The custom EditText class for numeric input interaction view. */
+class NumericInputInteractionView @JvmOverloads constructor(
+  context: Context,
+  attrs: AttributeSet? = null,
+  defStyle: Int = android.R.attr.editTextStyle
+  , private var placeholder: String = "Write the digit here."
+) : EditText(context, attrs, defStyle), InteractionAnswerRetriever {
+  init {
+    attributes()
+  }
+
+  constructor(context: Context?, placeholder: String) : this(context!!) {
+    this.placeholder = placeholder
+    attributes()
+  }
+
+  /** This function contains default attributes of [NumericInputInteractionView]. */
+  private fun attributes() {
+    setBackgroundResource(R.drawable.edit_text_background)
+    val paddingPixel = context.resources.getDimension(R.dimen.padding_8)
+    val density = resources.displayMetrics.density
+    val paddingDp = (paddingPixel * density).toInt()
+    setPadding(paddingDp, paddingDp, paddingDp, paddingDp)
+    hint = placeholder
+    keyListener = DigitsKeyListener.getInstance("0123456789.")
+    this.setEditTextMaxLength(length = 200)
+    setLongClickable(false)
```

Comment:
It can be modified to:

`isLongClickable = false`

Reply:
updated